src/lib/box.js

```diff
@@ -1037,7 +1037,7 @@ foam.CLASS({
   extends: 'foam.box.PromisedBox',
 
   requires: [
-    'foam.net.Socket',
+    'foam.net.web.Socket',
```

Comment:
So, this is definitely wrong. I pushed a fix that qualifies `Socket` and `SocketService` as `foam.net.node`, because that's the only environment they will work in.

@adamvy anything I'm missing there?